src/js/tech/html5.js

```diff
@@ -767,25 +767,7 @@ Html5.canControlPlaybackRate = function() {
  * @return {Boolean}
  */
 Html5.supportsNativeTextTracks = function() {
-  let supportsTextTracks;
-
-  // Figure out native text track support
-  // If mode is a number, we cannot change it because it'll disappear from view.
-  // Browsers with numeric modes include IE10 and older (<=2013) samsung android models.
-  // Firefox isn't playing nice either with modifying the mode
-  // TODO: Investigate firefox: https://github.com/videojs/video.js/issues/1862
-  supportsTextTracks = !!Html5.TEST_VID.textTracks;
-  if (supportsTextTracks && Html5.TEST_VID.textTracks.length > 0) {
-    supportsTextTracks = typeof Html5.TEST_VID.textTracks[0].mode !== 'number';
-  }
-  if (supportsTextTracks && browser.IS_FIREFOX) {
-    supportsTextTracks = false;
-  }
-  if (supportsTextTracks && !('onremovetrack' in Html5.TEST_VID.textTracks)) {
-    supportsTextTracks = false;
-  }
-
-  return supportsTextTracks;
+  return browser.IS_ANY_SAFARI;
```

Comment:
I agree with a move to doing emulated tracks wherever possible; I've been meaning to file an issue because, in order to support Descriptions tracks, you *have* to use emulated tracks even if the browser claims it supports text tracks, and having to switch to emulated tracks just *because* you want to add support for Descriptions tracks isn't going to be popular, if sites have become used to native track styles and controls. Emulated tracks allow the kind of cross-browser consistency, similar to control bar consistency, that is one of video.js' greatest strengths.